tests/test_cli.py

```diff
@@ -377,63 +350,46 @@ def test_run_non_ascii_file(self):
             locale.setlocale(locale.LC_ALL, 'C.UTF-8')
         except locale.Error:
             locale.setlocale(locale.LC_ALL, 'en_US.UTF-8')
+        self.addCleanup(locale.setlocale, locale.LC_ALL, loc)
```

Comment:
Can you revert this?
For correct testing and for reproductibility, it's better if the original locale is restored *at the end of this test*, not *at the end of all tests*.

Reply:
[`addCleanup()`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.addCleanup) happens after `tearDown()` - at the end of every test, not at the end of all.

Reply:
OK, thanks for the clarification!